src/Uno.Extensions.Reactive.Tests/Operators/Given_StateForEach.cs

```diff
@@ -113,14 +177,34 @@ public async Task When_UpdateStateAndCallbackFails_Then_CallbackInvokedOnNextUpd
 		await state.SetAsync(42, CT);
 		await state.SetAsync(3, CT);
 
+		//Test fails sometime because of time needed to process the exception
+		await Task.Delay(100);
```

Comment:
Hummm this is problematic and it indicates a threading / performance issue, everything should run sync in this test.

Reply:
Removed it.